tests/shared.ts

```diff
@@ -122,15 +121,14 @@ export const fixture = async (
     canSwap.programId
   );
 
-  const [wrappedTokenAccountAuthority, wrappedTokenAccountAuthorityBump] =
-    await PublicKey.findProgramAddress(
-      [
-        WRAPPED_TOKEN_OWNER_AUTHORITY_PDA_SEED,
-        canonicalMint.toBuffer(),
-        wrappedMint.toBuffer(),
-      ],
-      canSwap.programId
-    );
+  const [wrappedTokenAccountAuthority] = await PublicKey.findProgramAddress(
+    [
+      WRAPPED_TOKEN_OWNER_AUTHORITY_PDA_SEED,
+      canonicalMint.toBuffer(),
+      wrappedMint.toBuffer(),
+    ],
+    canSwap.programId
+  );
 
```

Comment:
We don't need to adjust the `8 + 68` below?

Reply:
I accidentally included this change in the previous PR, this byte count is correct